diff --git a/clr-rocm-5.6.0/rocclr/utils/flags.hpp b/clr-rocm-5.6.0/rocclr/utils/flags.hpp
index 186d059b..f6e7bb7d 100644
--- a/clr-rocm-5.6.0/rocclr/utils/flags.hpp
+++ b/clr-rocm-5.6.0/rocclr/utils/flags.hpp
@@ -245,7 +245,7 @@ release(bool, ROC_SYSTEM_SCOPE_SIGNAL, true,                                  \
         "Enable system scope for signals (uses interrupts).")                 \
 release(bool, ROC_SKIP_COPY_SYNC, false,                                      \
         "Skips copy syncs if runtime can predict the same engine.")           \
-release(bool, ROC_ENABLE_PRE_VEGA, false,                                     \
+release(bool, ROC_ENABLE_PRE_VEGA, true,                                     \
         "Enable support of pre-vega ASICs in ROCm path")                      \
 release(bool, GPU_FORCE_QUEUE_PROFILING, false,                               \
         "Force command queue profiling by default")                           \
--
2.40.1

